bindata/cert-manager-deployment/webhook/cert-manager-webhook-deployment.yaml

```diff
@@ -18,12 +18,16 @@ spec:
       app.kubernetes.io/name: webhook
   template:
     metadata:
+      annotations:
```

Comment:
Notes to my self,
Future TODO: test webhook metrics, update our ServiceMonitor docs with these metric paths.